reconcile/terraform_tgw_attachments.py

```diff
@@ -17,7 +17,7 @@
 QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
 
 
-def build_desired_state_tgw_attachments(clusters, ocm_map, settings):
+def build_desired_state_tgw_attachments(clusters, ocm_map, awsapi):
```

Comment:
nit: maybe not a bad time to add a type hint now, even if just for `awsapi`? Adding a single type hint will make Mypy actually check the cfunction.